src/component/Main/SingleArticleCard.jsx

```diff
@@ -27,7 +32,35 @@ export const SingleArticleCard = ({
       <p className="body__single_article">{body}</p>
       <div className="article__card__footer">
         <span>#{topic}</span>
+        <div className="article__icons">
+          <button
+            className="button__upvote"
+            onClick={() =>
+              handleSingleArticleUpvote(article_id, setUserUpvote, setErr)
+            }
+            disabled={userUpvote !== 0}
+          >
+            <i className="fa-regular fa-thumbs-up"></i>
+          </button>
+          <span className="article__number">
+          
```

Comment:
At the moment, the user loses their ability to change their vote after clicking each button - can you alter the condition which disables the button to fix this?

Reply:
Done white_check_mark